dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst

```diff
@@ -0,0 +1,60 @@
+#. On ``Output Name`` write `FilteredTemp`.
+#. On ``Sample Log`` drop-down menu select ``SampleTemp`` and hit on ``Plot`` button. Plot should update with the temperature log for the run vs. time.
+#. Moving vertical range sliders updates ``Minimum Value`` and ``Maximum Value`` text edits.
+#. Move the upper vertical range slider to be at approximately ``279.95`` degrees and the lower vertical range slider to be at approximately ``279.91`` degrees.
+#. Click on ``Filter`` button. That should generate several workspaces in the ADS. Two table workspaces ending with ``_info``, ``_splitters`` and a group workspace named ``FilteredTemp`` containing one
+   event workspace named ``FilteredTemp_0``, as well as a ``TOFCorrTable`` 2D workspace.
+#. Click second-mouse button on ``FilteredTemp_0`` and select ``Show Sample Logs``. On Sample Log Window, check the ``SampleTemp`` entry and make sure the temperature range is approximately
+   the same as selected with the sliders in the interface.
+#. Back to the filter events interface, click on ``Refresh`` button. The drop-down list should refresh with the available event workspaces on the ADS (``CNCS_7860_Event`` and ``FilteredTemp_0``).
+#. Select ``FilteredTemp_0`` and click on ``Use`` button. Plot should update accordingly.
+#. On output name write `FilteredTime`, on ``Starting Time`` text edit write ``80`` and then click ``Set``. On ``Stopping Time`` text edit write ``100`` and then click ``Set``.
+#. Select ``Filtered by Time`` tab and on ``Time Interval`` text edit write ``10``. Then click on ``Filter`` button. Two table workspaces ending with ``_info``, ``_splitters`` and a group workspace named ``FilteredTime`` containing two
+   event workspaces named ``FilteredTime_0`` and ``FilteredTime_1`` appear on ADS.
+#. Double-click on ``FilteredTemp_0_info`` table workspace. The table should contain two rows, indicating two time intervals of approximately 10 seconds each, and the corresponding workspace group index.
+#. Back to the filter events interface, without changing anything else from previous interaction, go to ``Advanced Setup`` tab and tick the ``Fast Log`` checkbox. Go back to ``Filter by Time`` and click
+   on ``Filter`` button again. Workspace ending in ``_splitters`` should be a 2D workspace, all other workspaces should remain the same.
+#. Now on ``Algorithms`` select ``FilterEvents`` algorithm and click on ``Execute``. As ``InputWorkspace`` select ``FilteredTemp_0`` workspace. On Splitter workspace select ``FilteredTemp_0_splitters``,
+   and on ``Information Workspace`` select ``FilteredTemp_0_info``. Check on ``GroupWorkspaces`` checkbox. And name the ``OutputWorkspaceBaseName`` as ``FilteredTimeRedux``.
+#. ``Run`` the algorithm. On the ADS, a new group event workspace named ``FilteredTimeRedux`` should appear. The two event workspaces that it contains should be equal to the two event workspace previuosly generated
+   through the interface, ``FilteredTime_0`` and ``FilteredTime_1``.
```

Comment:
I didn't get a group workspace here, I just got `FilteredTimeRedux_0` and `_1` output as separate workspaces. They were the same though.

Reply:
Mmm.. I'm getting the workspaces on a group. Did you tick on the `Group Workspaces` checkbox in the algorithm?

Reply:
Ah no, I don't think I did.

```suggestion
#. ``Run`` the algorithm with ``Group Workspaces`` checked. On the ADS, a new group event workspace named ``FilteredTimeRedux`` should appear. The two event workspaces that it contains should be equal to the two event workspace previuosly generated
   through the interface, ``FilteredTime_0`` and ``FilteredTime_1``.
```

Reply:
It's kind of already indicated in previous instruction?